### DISCUSSION TITLE
Templating sub button name causes icon to disappear

### DISCUSSION THREAD

**Discussion by schmuy** [Clooos/Bubble-Card#1005] [Questions about config, custom styles and templates]
Adding the following to the end of my styles in order to get the state of another sensor displayed causes the sub button icon to disappear.

${card.querySelector('.bubble-sub-button-1').innerText = hass.states['sensor.alllightsinsidecount'].state}

**Comment by MrBearPresident** [comment]
Not a bug you're doing it wrong. You have to use the` .bubble-sub-button-name-container` . 
Use:
```
${card.querySelector('.bubble-sub-button-1 .bubble-sub-button-name-container').innerText =  'Sometext'}

```


---

<sup>If this solves your issue, please mark this as solved and close it.</sup>
